gobblin-metastore/src/main/java/gobblin/metastore/FsStateStore.java

```diff
@@ -179,7 +192,7 @@ public State get(String storeName, String tableName, String stateId)
         State state = this.stateClass.newInstance();
```

Comment:
It's better to use `T state = ...` so you don't need to add `SuppressWarnings("unchecked")`